safe_transaction_service/account_abstraction/serializers.py

```diff
@@ -93,6 +78,39 @@ def _validate_signature(
             safe_signatures.append(safe_signature)
         return safe_signatures
 
+
+class SafeOperationSerializer(
+    SafeOperationSignatureValidatorMixin, serializers.Serializer
+):
+    nonce = serializers.IntegerField(min_value=0)
```

Comment:
nonce is uint256, maybe would be better set as decimal and the number of max digits. 
https://www.erc4337.io/docs/understanding-ERC-4337/user-operation

Reply:
Do you mean in this case to use `Uint256Field`? In that case I also agree and is in line with the other representations that we have for `nonce`.

Reply:
Yes, Uint256Field is a decimal field with the max_digits configured to fit to an unsigned 256 bits number.

Reply:
But you are pointing out the serializer, not the model. Serializer is alright with an `IntegerField`

Reply:
> But you are pointing out the serializer, not the model. Serializer is alright with an `IntegerField`

I would say that this is in line with the rest of the project 👍 

Maybe in the future we can discuss if `nonce` should be represented as string (same reasons as https://github.com/safe-global/safe-transaction-service/pull/2065#discussion_r1629487900). This would be a breaking change though.